eng/dockerfile-templates/runtime-deps/Dockerfile

```
@@ -9,6 +9,7 @@
     set isDebian to find(OS_ARCH_HYPHENATED, "Debian") >= 0 ^
     set isUbuntu to find(OS_ARCH_HYPHENATED, "Ubuntu") >= 0 ^
     set isAzureLinux to defined(match(OS_VERSION, "^cbl-mariner\d+\.\d+$")) || defined(match(OS_VERSION, "^azurelinux\d+\.\d+$")) ^
+    set isCblMariner2 to isAzureLinux && find(OS_VERSION_NUMBER, "2.0") >= 0 ^
```

This seems more logical to me:

```suggestion
    set isCblMariner2 to defined(match(OS_VERSION, "^cbl-mariner\d+\.\d+$")) ^
    set isAzureLinux to isCblMariner2 || defined(match(OS_VERSION, "^azurelinux\d+\.\d+$")) ^
```

Checking the Mariner version is unnecessary since there's only one version we provide.